### PR TITLE
Doc: Add credits. Reorganizing docs.

### PR DESCRIPTION
**Pull request by hungpham2511**

### CODE REVIEW

**Comment by jmirabel** [comment]
Thank you for the credits.

File: README.md [hunk 1]
```diff
@@ -42,44 +42,38 @@ To install depencidencies for development, replace the second command with:
...
-  [pytest](https://docs.pytest.org/en/latest/contents.html) for all
-  tests. Check out the test folder for more details.
+Pull Requests are welcomed! Go ahead to create a Pull Request and we
+will review your proposal!
+
+# Credits
+
+`toppra` was originally developed by [Hung
+PHam](https://hungpham2511.github.com/) (Eureka Robotics, former CRI
+Group) and [Pham Quang-Cuong](https://www.ntu.edu.sg/home/cuong/) with
```
Comment (quangounet) [inline README.md]: No need to include me as author. I haven't contributed a line of code ;)

File: README.md [hunk 2]
```diff
@@ -42,44 +42,38 @@ To install depencidencies for development, replace the second command with:
...
+major contributions from many contributors.  If you have taken part in
+developing and supporting the library, feel free to add your name to
+the list.
+- [Joseph Mirabel](https://github.com/jmirabel) (C+ API)
```
Comment (quangounet) [inline README.md:68]: C++

File: README.md [hunk 3]
```diff
@@ -42,44 +42,38 @@ To install depencidencies for development, replace the second command with:
...
-  contain unit tests. Note that `toppra` uses
-  [pytest](https://docs.pytest.org/en/latest/contents.html) for all
-  tests. Check out the test folder for more details.
+Pull Requests are welcomed! Go ahead to create a Pull Request and we
+will review your proposal!
+
+# Credits
+
+`toppra` was originally developed by [Hung
+PHam](https://hungpham2511.github.com/) (Eureka Robotics, former CRI
```
Comment (quangounet) [inline README.md]: Pham

File: README.md (hunk 1 above)
Comment (quangounet) [reply README.md]: Maybe I can be listed as one of the contributors (alongside with Joseph etc.) for "Suggestions on some aspects of the algorithm)

Comment (hungpham2511) [reply README.md]: The library enjoys many pointers from the TOPP library though :)

**Pull request merged**